apps/sse_contract_test_service/bin/service_api.openapi.yaml

```diff
@@ -75,8 +75,7 @@ components:
           type: string
         headers:
           type: object
-          additionalProperties:
```

Comment:
So, the generator works completely wrong in this case. It works 50% wrong with `true`. So the generated code has been manually edited.